assets/html/warner.js

```diff
@@ -37,7 +37,7 @@ function maybeAddWarning () {
         document.body.removeChild(div);
     });
     const href = window.documenterBaseURL + '/../' + window.DOCUMENTER_STABLE;
-    div.innerHTML = 'This is an old version of the documentation. <br> <a href="' + href + '">Go to the newest version</a>.';
+    div.innerHTML = 'This documentation is not for the latest version. <br> <a href="' + href + '">Go to the newest version</a>.';
```

Comment:
There is not `latest` and in the next sentence `newest`, perhaps it doesn't matter.

Reply:
```suggestion
    div.innerHTML = 'This documentation is not for the latest version. <br> <a href="' + href + '">Go to the latest documentation</a>.';
```
Dunno whether that reads better...

Reply:
I think it does read a little bit better, so let's go with that.